docs/docs/30-administration/11-agent-config.md

```diff
@@ -95,20 +95,89 @@ See [Conditionals Pipeline](/docs/usage/pipeline-syntax#step-when---conditional-
 
 Here is the full list of configuration options and their default variables. 
 
-```yaml
-    - WOODPECKER_SERVER=localhost:9000
-    - WOODPECKER_AGENT_SECRET=""
-    - WOODPECKER_USERNAME=x-oauth-basic
-    - WOODPECKER_LOG_LEVEL=""
-    - WOODPECKER_DEBUG_PRETTY=""
-    - WOODPECKER_DEBUG_NOCOLOR=true
-    - WOODPECKER_HOSTNAME=""
-    - WOODPECKER_PLATFORM="linux/amd64"
-    - WOODPECKER_FILTER=""
-    - WOODPECKER_MAX_PROCS=1
-    - WOODPECKER_HEALTHCHECK=true
-    - WOODPECKER_KEEPALIVE_TIME=10
-    - WOODPECKER_KEEPALIVE_TIMEOUT=time.Second * 20
-    - WOODPECKER_GRPC_SECURE=""
-    - WOODPECKER_GRPC_VERIFY=true
-```
+### `WOODPECKER_SERVER`
+> Default: `localhost:9000`
+
+Configures the Woodpecker server address.
+
+### `WOODPECKER_USERNAME`
+> Default: `x-oauth-basic`
+
+The gRPC username.
+
+### `WOODPECKER_AGENT_SECRET`
+> Default: empty
+
+A shared secret used by server and agents to authenticate communication. A secret can be generated by `openssl rand -hex 32`.
+
+### `WOODPECKER_DEBUG`
+> Default: `true`
+
+Enable agent debug mode.
+
+### `WOODPECKER_LOG_LEVEL`
+> Default: empty
+
+Configures the logging level. Possible values are `trace`, `debug`, `info`, `warn`, `error`, `fatal`, `panic`, `disabled` and empty.
+
+### `WOODPECKER_DEBUG_PRETTY`
+> Default: `false`
+
+Enable pretty-printed debug output.
+
+### `WOODPECKER_DEBUG_NOCOLOR`
+> Default: `true`
+
+Disable colored debug output.
+
+### `WOODPECKER_HOSTNAME`
+> Default: empty
+
+Configures the agent hostname.
+
+### `WOODPECKER_PLATFORM`
+> Default: `linux/amd64`
+
+Restrict builds by platform conditions.
+
+### `WOODPECKER_FILTER`
+> Default: empty
+
+Filter expression to restrict builds by label.
+
+Example: TODO
```

Comment:
I would not add it to the docs atm at all as it is will be changed in the next release anyways, so we don't have to deprecate it.